packages/devtools_server/lib/src/server.dart

```diff
@@ -174,10 +131,9 @@ Future<HttpServer> serveDevToolsWithArgs(
 /// Serves DevTools.
 ///
 /// `handler` is the [shelf.Handler] that the server will use for all requests.
-/// If null, [defaultHandler] will be used.
-/// Defaults to null.
+/// If null, [defaultHandler] will be used. Defaults to null.
+// Note: this method is used by the Flutter CLI.
```

Comment:
and dwds
https://github.com/dart-lang/webdev/blob/c625f2f9accfac87142cec8bf87c633c42be00b5/dwds/lib/src/servers/devtools.dart#L27

Reply:
👍 updated the comment